### DISCUSSION TITLE
User testing for Pytorch migration

### DISCUSSION THREAD

**Discussion by danielduckworth** [ludwig-ai/ludwig#1798] [General]
Hi everyone, just wondering if you want some user testing feedback for [v0.5rc2](https://github.com/ludwig-ai/ludwig/releases/tag/v0.5rc2)?

I mostly work with text data. Let me know if there is anything in particular I can help test.

**Comment by w4nderlust** [comment]
@danielduckworth that would be great! There is not something specific we need testing for, but if you have some models and data you use ludwig with, just replicating the same processes with 0.5 to make sure there are no issue would be extremely valuable.
Thank yo uvery much for offering!
